src/ffbwrapper.c

```diff
@@ -56,9 +60,17 @@ static int enable_features_hack = 0;
 static int enable_force_inversion = 0;
 static int ignore_set_gain = 0;
 static int enable_offset_fix = 0;
+static int enable_throttling = 0;
 static FILE *log_file = NULL;
 static char report_string[1024];
 static short last_effect_used = 16;
+static char effect_is_pending[64];
```

Comment:
since the size of this is used in so many places, probably best to keep that in a constant or a `#define` in case it's changed. Right now it would be very easy to miss it in the other locations.

Reply:
Yes, that would be best.